lib/functions/sketch.R

```diff
@@ -0,0 +1,23 @@
+# In this file we're trying out some code
+library(tidyverse)
+library(ggplot2)
+library(gridExtra) # For arranging multiple plots
+
+# Load data
+df_measures <- readr::read_csv(
+  here::here("output", "report", "conditions_measures.csv")
+)
+
+# Develop plotting function 
+p <- ggplot(df_measures, aes(x = interval_end, y = numerator, color = measure, group = measure)) +
+  geom_line() +
+  labs(
+    title = paste("Number of Consultations for each Pharmacy First Clinical Condition per month"),
+    x = "Date",
+    y = "Number of Consultations",
+    color = "Condition"
+  ) +
+  ylim(0, NA)
+
+source(here::here("lib", "functions", "graph_func.R"))
```

Comment:
It's just the sketch file so doestn matter if there are errors, but we'd need to change the file name here if we want to work: 
```suggestion
source(here::here("lib", "functions", "function_plot_measures.R"))
```